examples/align-advanced/rs-align-advanced.cpp

```diff
@@ -158,14 +172,15 @@ void render_slider(rect location, float& clipping_dist)
 
     //Render the vertical slider
     ImGui::Begin("slider", nullptr, flags);
-    ImGui::PushStyleColor(ImGuiCol_FrameBg, ImColor(215.f / 255, 215.0f / 255, 215.0f / 255));
-    ImGui::PushStyleColor(ImGuiCol_SliderGrab, ImColor(215.f / 255, 215.0f / 255, 215.0f / 255));
-    ImGui::PushStyleColor(ImGuiCol_SliderGrabActive, ImColor(215.f / 255, 215.0f / 255, 215.0f / 255));
+    ImGui::PushStyleColor(ImGuiCol_FrameBg, { 0.2f / 255, 0.3f / 255, 0.7f / 255 ,1});
```

Comment:
Can you remind me why
215/255 is changed to 0.2/255?

Reply:
ImGui::PushStyleColor should get ImVec4 format for the colors and not ImColor obj.
I checked and its possible to get the ImVec4 value out of ImColor so I switched it to that.
The colors look the same as they were before :)